certstore_windows.go

```diff
@@ -441,7 +457,8 @@ func (wpk *winPrivateKey) cngSignHash(hash crypto.Hash, digest []byte) ([]byte,
 }
 
 // capiSignHash signs a digest using the CryptoAPI APIs.
-func (wpk *winPrivateKey) capiSignHash(hash crypto.Hash, digest []byte) ([]byte, error) {
+func (wpk *winPrivateKey) capiSignHash(opts crypto.SignerOpts, digest []byte) ([]byte, error) {
+	hash := opts.HashFunc()
```

Comment:
Since CAPI does not support the RSA-PSS algorithm, would it make sense to check if `capiSignHash` was given an `opts` that successfully casts to `rsa.PSSOptions` and return an error?

Reply:
I see what you're saying, but will the PSS flow ever get to this func?